library/src/main/java/com/bumptech/glide/manager/RequestManagerRetriever.java

```diff
@@ -133,6 +149,136 @@ public RequestManager get(Fragment fragment) {
     }
   }
 
+  public RequestManager get(View view) {
+    if (view.getContext() == null) {
+      throw new IllegalArgumentException("Unable to obtain a request manager for a view without"
+          + " a Context");
+    }
+    Activity activity = findActivity(view.getContext());
+    // The view might be somewhere else, like a service.
+    if (activity == null) {
+      return get(view.getContext().getApplicationContext());
+    }
+
+    // Support Fragments.
+    if (activity instanceof FragmentActivity) {
+      Fragment fragment = findSupportFragment(view, (FragmentActivity) activity);
+      if (fragment == null) {
+        return get(activity);
+      }
+      return get(fragment);
+    }
+
+    // Standard Fragments.
+    android.app.Fragment fragment = findFragment(view, activity);
```

Comment:
There's no help of erasure here. This should crash with a ClassNotFoundException on older devices if the activity is not a `FragmentActivity`.